### ISSUE TITLE
GGMax Bug - It became an indelible square hole in the level where I drop out of the world.

### ISSUE THREAD

**Issue by Jackdanielsvs68** [TheGameCreators/GameGuruRepo#5869]
It became an indelible square hole in the level where I drop out of the world. At the beginning of the test, it was there under the starting position, and I can't make it disappear. If I change the terrain, it will still stay there.

![Képernyőfelvétel (293)](https://github.com/user-attachments/assets/e901e8a2-9330-441d-a6fd-288fc72154af)
![Képernyőfelvétel (295)](https://github.com/user-attachments/assets/11625c71-549a-445e-bfe0-d1742ba0fe8d)
![Képernyőfelvétel (296)](https://github.com/user-attachments/assets/b832404b-e49d-48c1-9c88-fe0eb2b48497)
![Képernyőfelvétel (297)](https://github.com/user-attachments/assets/2de63c31-29dc-4a48-a360-b30c3e920ca8)

**Comment by korodai** [comment]
That happened to me too. It turned out an EBE structure had somehow been placed beneath the map, creating a large hole. Once I located and deleted the structure, the issue was resolved.

**Comment by Jackdanielsvs68** [comment]
There really was a building there. I had to find it, position it, and delete all its accessories one by one, doors, windows, lights. But there were also a few weapons left there that I never put there. Even if I delete them, they will return there even after saving.

**Comment by Jackdanielsvs68** [comment]
![Képernyőfelvétel (298)](https://github.com/user-attachments/assets/d1d5c72a-8e1b-48a2-876f-54fabf826244)
![Képernyőfelvétel (299)](https://github.com/user-attachments/assets/bbc63309-510c-4856-ac25-beb3d0079568)
![Képernyőfelvétel (300)](https://github.com/user-attachments/assets/4be0db53-92af-4fbe-8f4a-55905b1eab52)
![Képernyőfelvétel (301)](https://github.com/user-attachments/assets/4e97c420-4a91-4d97-9335-4989e6629d63)

**Comment by Kitakazi** [comment]
Yes those weapons are auto generated, they are from your collectable item list. They are auto-generated by default for some reason. The files are in your documents gameguru folder. Under files/projectectbank/ 
Inside your project is the file collection - items.tsv 
I've heard if you delete the items from your level, save, then go to that file and remove the items from the list then its not supposed to auto-generate them again. However no matter how many times I remove them they continue to auto generate for me so I'm not sure if its a bug or what. Very annoying.

**Comment by Kasseyus** [comment]
Did this occur after placing a BE structure into the world, and if so, does it happen every time you place a building?

(the auto-gen weapons issue exists as a separate ticket)

**Comment by korodai** [comment]
It only happened once to me and that was when I moved a BE structure and somehow it ended up under the map.

**Comment by Jackdanielsvs68** [comment]
There is a BE structure on the level, but not there, but in another place.. The hole appeared out of nowhere with the start position above it. I saved the level after a change and started the test mode. I emerged underwater in the hole, which was not there before.

**Comment by Jackdanielsvs68** [comment]
> Megtörtént ez azután, hogy egy BE-szerkezetet elhelyeztek a világban, és ha igen, akkor minden alkalommal, amikor egy épületet elhelyeztek?
> 
> (az auto-gen fegyverek kérdése külön jegyként létezik)

I encountered this error a long time ago, when GGMAX was still in early access. It hasn't happened to me since then, just now.

**Comment by LeeBamberTGC** [comment]
NOTE: Just going through the older 2024 bug reports marked WAITING FOR MORE INFO.  Feel free to add new info here using the latest DEVTEST build. If nothing new is posted, I will assume it cannot be reproduced with the newer updates and close this one in the near future.

**Comment by Jackdanielsvs68** [comment]
The phenomenon did not improve at that level, but I did not encounter it again elsewhere.
